tests/testthat/test_barber_smooth.R

```diff
@@ -0,0 +1,12 @@
+test_that("The function runs", {
+  vec <- c(1, 7, 5, 6, 19, 10)
+  smoothed <- barber_smooth(vec, n_neighbors = 2, times = 1)
+
+  expect_equal(length(smoothed), 6)
+  expect_equal(class(smoothed), "matrix")
```

Comment:
@aucarter do you know if this should be converting the result back to a vector?

Reply:
it probably should be a vector, but is not utilized in any ways where that matters. i am going to leave it as is for now.